hexrd/ui/indexing/fit_grains_results_dialog.py

```diff
@@ -442,6 +445,12 @@ def update_range_constraints(self):
             w1.setMaximum(w2.value())
             w2.setMinimum(w1.value())
 
+    def set_glyph_size(self, reset=True):
```

Comment:
Maybe this should be called `reset_glyph_size`? With the `reset` argument being called `update_plot`?

Reply:
Sounds good, makes it clearer. I've updated the name of the function and parameter.